src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs

```diff
@@ -359,7 +359,17 @@ public string GetValue(string section, string key, bool isPath = false)
                 curr = curr._next;
             }
 
-            return ret;
+            return ApplyEnvironmentTransform(ret);
+        }
+
+        private string ApplyEnvironmentTransform(string configValue)
+        {
+            if (String.IsNullOrEmpty(configValue))
```

Comment:
I was matching the casing existing in the file, e.g. https://github.com/TravisTheTechie/NuGet.Client/blob/issue-1852-eval-env-variables-in-config-values/src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs#L59, should I go ahead an update the other references to match or just leave it with my changes?